content/en/ecosystem.md

```diff
@@ -18,6 +18,7 @@ We are happy and proud to have you all as part of our community! :sparkling_hear
 
 | Vendor      | Product / Service       | Link                                                               |
 |-------------|-------------------------|--------------------------------------------------------------------|
+| AWS         | EKS Anywhere            | [Documentation](https://anywhere.eks.amazonaws.com/docs/)          |
```

Comment:
```suggestion
| AWS         | EKS Anywhere            | [Documentation](https://anywhere.eks.amazonaws.com/docs/tasks/cluster/cluster-flux//)          |
```

Reply:
Had the same thought but then decided to align the link with the other ones that link to the top-level docs but I'm happy to deep-link here.